ui/src/app/app-routing.tsx

```diff
@@ -1,225 +1,84 @@
-const UserAdminPage = fp.flow(withRouteData, withRoutingSpinner)(AdminUser);
-const UsersAdminPage = fp.flow(withRouteData, withRoutingSpinner)(AdminUsers);
-const UserAuditPage = fp.flow(withRouteData, withRoutingSpinner)(UserAudit);
 const UserDisabledPage = fp.flow(withRouteData, withRoutingSpinner)(UserDisabled);
-const WorkspaceWrapperPage = fp.flow(withRouteData, withRoutingSpinner)(WorkspaceWrapper);
-const WorkspaceAdminPage = fp.flow(withRouteData, withRoutingSpinner)(AdminWorkspace);
-const WorkspaceAuditPage = fp.flow(withRouteData, withRoutingSpinner)(WorkspaceAudit);
-const WorkspaceEditPage = fp.flow(withRouteData, withRoutingSpinner)(WorkspaceEdit);
-const WorkspaceLibraryPage = fp.flow(withRouteData, withRoutingSpinner)(WorkspaceLibrary);
-const WorkspaceListPage = fp.flow(withRouteData, withRoutingSpinner)(WorkspaceList);
-const WorkspaceSearchAdminPage = fp.flow(withRouteData, withRoutingSpinner)(AdminWorkspaceSearch);
 
 interface RoutingProps {
   onSignIn: () => void;
   signIn: () => void;
+  subscribeToInactivitySignOut: () => void;
+  signOut: () => void;
 }
 
-export const AppRoutingComponent: React.FunctionComponent<RoutingProps> = ({onSignIn, signIn}) => {
+export const AppRoutingComponent: React.FunctionComponent<RoutingProps> = ({onSignIn, signIn, subscribeToInactivitySignOut, signOut}) => {
   const {authLoaded = false} = useStore(authStore);
 
   return authLoaded && <React.Fragment>
     {/* Once Angular is removed the app structure will change and we can put this in a more appropriate place */}
     <NotificationModal/>
     <AppRouter>
-      <AppRoute
-          path='/cookie-policy'
-          component={() => <CookiePolicyPage routeData={{title: 'Cookie Policy'}}/>}
-      />
-      <AppRoute
-          path='/login'
-          component={() => <SignInPage routeData={{title: 'Sign In'}} onSignIn={onSignIn} signIn={signIn}/>}
-      />
-      <AppRoute
-          path='/session-expired'
-          component={() => <SessionExpiredPage routeData={{title: 'You have been signed out'}} signIn={signIn}/>}
-      />
-      <AppRoute
-          path='/sign-in-again'
-          component={() => <SignInAgainPage routeData={{title: 'You have been signed out'}} signIn={signIn}/>}
-      />
-      <AppRoute
-          path='/user-disabled'
-          component={() => <UserDisabledPage routeData={{title: 'Disabled'}}/>}
-      />
-      <ProtectedRoutes guards={[signInGuard]}>
-        <AppRoute path='/access-renewal' component={() => <AccessRenewalPage routeData={{title: 'Access Renewal'}}/>}
-        />
-        <ProtectedRoutes guards={[expiredGuard]}>
-          <AppRoute
-            path='/'
-              component={() => <HomepagePage routeData={{title: 'Homepage'}}/>}
-          />
-        </ProtectedRoutes>
-        <AppRoute
-            path='/admin/banner'
-            component={() => <AdminBannerPage routeData={{title: 'Create Banner'}}/>}
-        />
+      <Switch>
```

Comment:
nit: I would put a comment here just to mention that this works because `Switch` duck-types the child components, looking for a `path` prop